Python/specialize.c

```diff
@@ -2556,6 +2609,7 @@ binary_op_extended_specialization(PyObject *lhs, PyObject *rhs, int oparg,
 
     LOOKUP_SPEC(compactlong_float_specs, oparg);
     LOOKUP_SPEC(float_compactlong_specs, oparg);
+    LOOKUP_SPEC(compactlongs_specs, oparg);
```

Comment:
Why three tables, rather than one?

Reply:
A single table would need to have a list of (guard, action) pairs for each OP. So it's a table of tables. Same thing basically.